docs/governance/governance.mdx

```diff
@@ -17,31 +17,34 @@ import { TopLedgerEmbed } from '@site/src/theme/TopLedgerEmbed'
 
 Helium governance exists to manage the process by which the network can be modified, whether in its
 technical architecture, the token's economic model, or meta-governance changes. It is governed by
-the global community of owners of Helium tokens ([HNT](/tokens/hnt-token),
-[MOBILE](/tokens/mobile-token), or [IOT](/tokens/iot-token)). These token owners include Hotspot
-deployers, network users, wireless network operators, hardware manufacturers, and more. Tokens must
-be locked for voting on network changes, ensuring long-term stakeholder alignment.
-
-Anyone can submit a Helium Improvement Proposal ([HIP](/governance/hip)) as a first step to
-modifying the network. All HIPs are published in the public Helium GitHub repository, then discussed
-and debated in public forums, including the Helium community Discord server, and on monthly
-community calls. Any Helium token owner is eligible to vote on HIPs within the domain of that
-specific token. To vote, a Helium token owner must first lock their tokens for a period of time, up
-to four years, which grants the owner voting power determined by the number of locked tokens and the
-lock duration.
-
-As Helium is a network of networks, there are currently three different jurisdictions for
-governance:
-
-1. Locking HNT allows for voting on HIPs governing the <u>Helium network as a whole</u>.
-1. Locking IOT enables voting on HIPs governing the <u>IoT subnetwork</u>.
-1. Locking MOBILE enables voting on HIPs governing the <u>mobile subnetwork</u>.
+the global community of owners of Helium ([HNT](/tokens/hnt-token) tokens.
+
+These token owners include Hotspot deployers, network users, wireless network operators, hardware 
+manufacturers, network mappers and more. Tokens must be locked for voting on network changes, ensuring 
+long-term stakeholder alignment.
+
+
+Since the implementation of [HIP-141](https://github.com/helium/HIP/blob/main/0141-single-token-governance-and-release-proposals.md#votes-with-code) the governance process has changed. The Helium Community 
```

Comment:
I think we can drop the "Since the implementation of HIP-141..." and focus on the current implementation.